history-network.md

```diff
@@ -147,14 +147,38 @@ each receipt/transaction and re-rlp-encode it, but only if it is a legacy transa
 
 #### Block Header
 
+
+```
+# Content types
+
+AccumulatorProof = Vector[Bytes32, 15]
```

Comment:
I see you've added this as a possible TODO in the main PR description.  Yes, I do believe adding a description for how this proof **must** be constructed is necessary here.  We are going to have a bunch of 32 byte trie nodes and we need a defined order for how they are placed into this list.

I believe the specification for the proof needs to be relatively specific in the following ways

- disallow extraneous trie nodes that aren't necessary for the proven data.
- required ordering of trie nodes

Reply:
I was planning on linking to the SSZ proofs of consensus-specs (https://github.com/ethereum/consensus-specs/blob/dev/ssz/merkle-proofs.md), as we use what is defined there. And that is basically: proof -> ordered from lowest digest (leave sibling) to root-1. Not sure if I need to repeat that.

Was planning on writing down extra information of what the proof is build against (epoch accumulator) and verified with leave + proof + master accumulator digest.

Reply:
:+1: on referencing the external spec.  No need to duplicate.  I didn't dig into the spec thoroughly enough, but I would like to be sure that the order of the trie nodes is well defined and I didn't immediately see whether it specifies this